### PR TITLE
readme cleanup

### PR DESCRIPTION
**Pull request by Esshahn**
cleanup while RTFM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -244,7 +243,7 @@ Missing Features
 
 * VERA
 	* Does not support the "CURRENT_FIELD" bit
-	* Does not sprite collisions
+	* Does not support sprite collisions
 	*	* Interlaced modes (NTSC/RGB) don't render at the full horizontal fidelity
```
Comment (greg-king5) [inline README.md]: There should be only one `*` in that line.

File: README.md [hunk 2]
```diff
@@ -304,7 +303,7 @@ Release Notes
 * GIF recording can now be controlled from inside the machine [Randall Bohn]
 * Debugging
 	* Major enhancements to the debugger [kktos]
-	* `-echo` will now encode non-printable characters like this: \X93 for CHR$(93), -bas as 	well as pasting accepts this convention again
+	* `-echo` will now encode non-printable characters like this: \X93 for CHR$(93), `-bas` as 	well as pasting accepts this convention again
 	* `-echo raw` for the original behavior
 	* `-echo iso` to for correct character encoding in ISO mode
```
Comment (greg-king5) [inline README.md]: Remove the word "to".

File: README.md [hunk 3]
```diff
@@ -324,41 +323,41 @@ Release Notes
...
-* Ctrl + V pastes the clipboard as keypresses
-* -bas file.txt loads a BASIC program in ASCII encoding
-* -echo prints all BASIC/KERNAL output to the terminal, use it with LIST to convert a BASIC program to ASCII
-* -run acts like -prg, but also autostarts the program
+* keyboard shortcuts work on Windows/Linux: `Ctrl + F/R/S/V`
+* `Ctrl + V` pastes the clipboard as keypresses
+* `-bas file.txt` loads a BASIC program in ASCII encoding
+* `-echo` prints all BASIC/KERNAL output to the terminal, use it with LIST to convert a BASIC program to ASCII
+* `-run` acts like `-prg`, but also autostarts the program
 * JMP $FFFF and SYS 65535 exit the emulator and save memory the host's storage
```
Comment (greg-king5) [inline README.md]: That line should say,
```
* `JMP $FFFF` and `SYS 65535` exit the emulator, and save memory into the host's storage
```

**Comment by nigels-com** [comment]
Looks good to me.

**Comment by mist64** [comment]
Thanks!

**Pull request merged**